sycl/test-e2e/VirtualMem/vector_with_virtual_mem.cpp

```diff
@@ -142,7 +142,7 @@ int main() {
   // To better test the functionality, try to allocate below the granularity
   // but enough to require more memory for some iterations.
   size_t SizeIncrement = 11;
-  size_t MinSizeGran = syclext::get_minimum_mem_granularity(Q) / sizeof(int);
+  size_t MinSizeGran = syclext::get_mem_granularity(Q) / sizeof(int);
```

Comment:
We should pass mode here, right?

Reply:
We could, but for this test it doesn't really matter if we use minimum or recommended.

Reply:
ok)